README.md

```diff
@@ -99,6 +116,16 @@ Refering $SPARK_HOME to the Spark installation directory. This library has been
 | kinesis.client.numRetries |     3 |  Maximum Number of retries for Kinesis API requests  |
 | kinesis.client.retryIntervalMs |     1000 |  Cool-off period before retrying Kinesis API  |
 
+## Kinesis Sink Configuration
+ Option-Name        | Default-Value           | Description  |
+| ------------- |:-------------:| -----:|
+| sink.streamName   | - | Name of the stream in Kinesis to write to|
```

Comment:
nit: Shall we drop `sink` before the option name. it will be consistent with other configuration name